src/TaskTracker/components/TaskTrackerCard/TaskTrackerCard.js

```diff
@@ -82,21 +82,46 @@ const TaskTrackerCard = ({ title, table, year, affiliation, code, time, complete
             if (time == '0') {
               if (complete_col == 'empty_bag_wt' || complete_col == 'fresh_biomass_wt') {
                 for (let item = 0; item < response.length; item++) {
-                  if (response[item].protocols.decomp_biomass == 0) {
+                  if (
+                    response[item].protocols.decomp_biomass == null ||
+                    response[item].protocols.decomp_biomass == 0 ||
+                    response[item].protocols.decomp_biomass == -999
+                  ) {
                     response.splice(item, 1);
                     item--;
                   }
                 }
               }
             } else if (time != '0' || time != '') {
               for (let item = 0; item < response.length; item++) {
-                if (response[item].protocols.decomp_biomass == 0) {
+                if (
+                  response[item].protocols.decomp_biomass == null ||
+                  response[item].protocols.decomp_biomass == 0 ||
+                  response[item].protocols.decomp_biomass == -999
+                ) {
                   response.splice(item, 1);
                   item--;
                 }
               }
             }
           }
+          if (
+            title == 'Fresh weight' &&
```

Comment:
should this be for the yield chips @brianwdavis? i'm not positive i understood the issue. if `protocols.cash_crop_yield = 0 or -999 or null` we want to not show any of the 4 yield chips right? this current logic is hiding the biomass chips